cmd/erasure-server-pool.go

```diff
@@ -1791,6 +1881,13 @@ func (z *erasureServerPools) PutObjectPart(ctx context.Context, bucket, object,
 		return PartInfo{}, err
 	}
 
+	// Add the object part to the cache
+	z.multipartUploadsCache.Set(uploadID, MultipartUpload{
+		Bucket:   bucket,
+		Object:   object,
+		UploadID: uploadID,
+	})
+
```

Comment:
the multipart uploadID must be set in newMultipartUpload() and 
removed from the map when 

- CompleteMultipart happens
- or AbortMultipart happens

We also need a peer call where we can notify our peers that they need to delete their local uploadID cache in memory.

Reply:
It should never be set for every PutObjectPart()

Reply:
oh ok, will change accordingly.